apiary.apib

```diff
@@ -4764,7 +4767,7 @@ Gets configuration rows.
             ]
 
 
-## Get Configuration Row Detail [/v2/storage/components/{component_id}/configs/{config_id}/rows/{row_id}]
+## Manage Configuration Rows [/v2/storage/components/{component_id}/configs/{config_id}/rows/{row_id}]
```

Comment:
Tady jde o to, že tam byla duplicitní URL v té sekci ## 

Mít extra rozdělené nadpisem "get configuration row detail" a "manage configuration rows" mi přišlo zbytečné, tak je to pod jedním nadpisem a jednou URL.

Reply:
Este ked na to pozeram tak si hovorim ze by sa mohlo spojit `Create or List Configuration Rows` do toho `Manage Configuration Rows`  
![Screenshot 2020-05-29 at 09 27 07](https://user-images.githubusercontent.com/6448364/83233239-b73ede00-a18e-11ea-9202-8a44726528f8.png)
ze mi pride ze je to vsetko `Manage`

Reply:
To sice jo, ale jsou to různé URL - create/list nemá na konci ID, ty manage ano...

Reply:
Ok tak to asi nechajme